app/Model/Server.php

```diff
@@ -6935,6 +6947,95 @@ private function generateServerSettings()
                     'null' => true
                 ]
             ),
+            'SimpleBackgroundJobs' => [
+                'branch' => 1,
+                'enabled' => [
+                    'level' => 2,
+                    'description' => __('Enables or disables background jobs with Supervisor backend.'),
+                    'value' => false,
+                    'test' => 'testBool',
+                    'type' => 'boolean'
+                ],
+                'redis_host' => [
+                    'level' => 2,
+                    'description' => __('The host running the redis server to be used for background jobs.'),
+                    'value' => '127.0.0.1',
+                    'test' => 'testForEmpty',
+                    'type' => 'string'
+                ],
+                'redis_port' => [
+                    'level' => 2,
+                    'description' => __('The port used by the redis server to be used for background jobs.'),
+                    'value' => 6379,
+                    'test' => 'testForNumeric',
+                    'type' => 'numeric'
+                ],
+                'redis_database' => [
+                    'level' => 2,
+                    'description' => __('The database on the redis server to be used for background jobs. If you run more than one MISP instance, please make sure to use a different database on each instance.'),
+                    'value' => 1,
+                    'test' => 'testForNumeric',
+                    'type' => 'numeric'
+                ],
+                'redis_password' => [
+                    'level' => 2,
+                    'description' => __('The password on the redis server (if any) to be used for background jobs.'),
+                    'value' => '',
+                    'test' => null,
+                    'type' => 'string',
+                    'redacted' => true
+                ],
+                'redis_namespace' => [
+                    'level' => 2,
+                    'description' => __('The namespace to be used for the background jobs related keys.'),
+                    'value' => '',
+                    'test' => null,
+                    'type' => 'string'
+                ],
+                'max_job_history_ttl' => [
+                    'level' => 2,
+                    'description' => __('The time in seconds the job statuses history will be kept.'),
+                    'value' => '',
+                    'test' => 'testForNumeric',
+                    'type' => 'numeric'
+                ],
+                'track_status' => [
+                    'level' => 2,
+                    'description' => __('Default behavior for tracking jobs statuses.'),
```

Comment:
I don't understand what this setting means :/

Reply:
Track in cakeresque allows us to request the status of a running job by pid. This should always be true from a MISP perspective and doesn't need a setting.

Reply:
removed the setting, the parameter is kept in BackgroundJobsTool::enqueue() for compatibility with CakeResque jobs.